components/navigation.js

```diff
@@ -406,6 +406,46 @@ NavigationHrefItem.propTypes = {
   isNarrowNav: PropTypes.bool,
 };
 
+/**
+ * Renders a single navigation item that links to an external site.
+ */
+function NavigationAnchorItem({
+  id,
+  href,
+  navigationClick,
+  isChildItem = false,
+  isNarrowNav = false,
+  children,
+}) {
+  const cssClasses = navigationClasses(isNarrowNav, isChildItem);
+
+  return (
+    <li>
+      <a
```

Comment:
Since it is an external site, should it open in a new tab instead of the original tab?

Reply:
I usually think so, but the igvf.org site links to our page without opening a new tab, so I felt maybe it makes sense to return to that page without opening a new tab.

Reply:
Oh wait, you’re talking about the catalog. Yeah, I thought about that, but I really wasn’t sure.